browser/net/url_context.cc

```diff
@@ -105,9 +105,11 @@ std::shared_ptr<brave::BraveRequestInfo> BraveRequestInfo::MakeCTX(
   // |AddChannelRequest| provides only old-fashioned |site_for_cookies|.
   // (See |BraveProxyingWebSocket|).
   if (ctx->tab_origin.is_empty()) {
-    ctx->tab_origin = brave_shields::BraveShieldsWebContentsObserver::
-                          GetTabURLFromRenderFrameInfo(ctx->frame_tree_node_id)
```

Comment:
It seems like you can completely remove `BraveShieldsWebContentsObserver::GetTabURLFromRenderFrameInfo()` from the repository now, since this was apparently the only caller.